pkg/cmd/pipelinerun/describe_test.go

```diff
@@ -315,6 +315,39 @@ func TestPipelineRunDescribe_failed(t *testing.T) {
 	golden.Assert(t, actual, fmt.Sprintf("%s.golden", t.Name()))
 }
 
+func TestPipelineRunDescribe_last_no_PipelineRun_present(t *testing.T) {
+	clock := clockwork.NewFakeClock()
```

Comment:
This isn't needed for this test. Remove all references to clock.

Reply:
done